doc/source/rest.j2

```diff
@@ -551,10 +551,80 @@ argument, and in this case the second argument passed is the value, or it.
 The operators or (`or` or `∨`), and (`and` or `∧`) and `not` are also
 supported, and take a list of arguments as parameters.
 
+.. _aggregates:
+
+Aggregates: on the fly, measurements modification and aggregatation
+===================================================================
+
+Gnocchi allows to do on-the-fly aggregation and modification of already
+aggregated data of |metrics|.
+
+It can also be done by providing the list of |metrics| to aggregate:
+
+{{ scenarios['get-aggregates-by-metric-ids']['doc'] }}
+
+This example computes the mean aggregates with `all` metrics listed in
+`metrics` and then multiple it by `4`.
+
+.. Note:
+
+   `operations` can also be passed as a lisp string, for this example:
+   `"operations": "(aggregate mean (metric all))"`
+
+Operation between metrics can also be done, like:
+
+{{ scenarios['get-aggregates-between-metrics']['doc'] }}
+
+.. Partial list of supported maths/aggregation::
```

Comment:
curious, what's the '..' for?

Reply:
I have though I will have a box like "Note:", but instead the content is hidden.... I removed them.